### ISSUE TITLE
ActionGroup: When  overflowMode collapse is used, resulting menubutton with ellipsis needs a better label

### ISSUE THREAD

**Issue by dineshy87** [adobe/react-spectrum#4129]
# 🐛 Bug Report 
When OverflowMode: collapse results in Actiongroups behaving like menu buttons,  the resulting menubutton seems to have an aria-label=“…”. This is announced as "ellipsis" when voiceOver is used. And on windows, NVDA and narrator consider it as having no label and announces nothing. Refer to this [storybook example](https://reactspectrum.blob.core.windows.net/reactspectrum/41e26fa0f01ee9ab6e5c21edf6f7bb5d498bf40f/storybook/index.html?path=/story/actiongroup--overflowmode-collapse&providerSwitcher-express=false&providerSwitcher-theme=light&providerSwitcher-toastPosition=bottom)

Note: This effects ActionBar as well and is more of an issue on mobile because actiongroup is almost always collapsed on mobile. 

## 🤔 Expected Behavior

Instead of aria-label="...", **_more_** may be a better label for these menubuttons or something else. In cases where entire toolbar can be collapsed to one button, the menubutton must take that the label of toolbar itself. (I see that it's already happening in [this example](https://reactspectrum.blob.core.windows.net/reactspectrum/41e26fa0f01ee9ab6e5c21edf6f7bb5d498bf40f/storybook/index.html?path=/story/actiongroup--overflowmode-collapse-selection&providerSwitcher-express=false&providerSwitcher-theme=light&providerSwitcher-toastPosition=bottom))

## 😯 Current Behavior

Collapsed menubutton has aria-label="..." which is completely ignored by some screen readers. And therefore is not an ideal label for the button.

## 💻 Code Sample

Issue can be observed on these [storybook examples](https://reactspectrum.blob.core.windows.net/reactspectrum/41e26fa0f01ee9ab6e5c21edf6f7bb5d498bf40f/storybook/index.html?path=/story/actiongroup--overflowmode-collapse&providerSwitcher-express=false&providerSwitcher-theme=light&providerSwitcher-toastPosition=bottom) by testing using NVDA on Firefox/Edge. 

You can use https://assistivlabs.com/ to verify this if you do not have a windows machine. 

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   | 3|
| Browser          |  Firefox & Edge (at least) |
| Operating System | Windows

**Comment by snowystinger** [comment]
Storybook is not our documentation. It's just for internal testing. Our docs page appears to have a better label https://react-spectrum.adobe.com/react-spectrum/ActionMenu.html#actionmenu
Closing, if there are any in the docs pages which need better labels, we can reopen.

**Comment by snowystinger** [comment]
Ah, I see. The ellipsis does appear in the docs if you get the ActionGroup to partially collapse. The aria-label is only passed through to the ActionButton for the menu if every item is hidden.
We should probably have a better/localized string instead of the '…'
https://github.com/adobe/react-spectrum/blob/cfd890edf73783cbea0b94316d69f4bdced5ed8a/packages/%40react-spectrum/actiongroup/src/ActionGroup.tsx#L390
